### PR TITLE
Add .cairo-coverage-ignore documentation

### PR DESCRIPTION
**Pull request by ksew1**
**Stack**:
- #111 ⬅
- #110
- #109
- #108

⚠️ *Part of a stack created by [spr](https://github.com/ejoffe/spr). Do not merge manually using the UI - doing so may have unexpected results.*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,6 +35,12 @@ curl -L https://raw.githubusercontent.com/software-mansion/cairo-coverage/main/s
 
 ## Usage
 
+### `.cairo-coverage-ignore` File
+
+You can create a `.cairo-coverage-ignore` file in the root of your project to specify the files or directories that you
+want to exclude from the coverage report. The file should contain a list of paths to be ignored, with one path per line,
+similar to a `.gitignore` file.
```
Comment (THenry14) [inline README.md:42]: we could also mention that regexes work there, pretty significant info

Comment (ksew1) [reply README.md:42]: Added 😄

**Comment by ksew1** [comment]
Seems that when we use forge from registry sometimes CI is failing:

```     Blocking waiting for file lock on package cache
warn: failed to open local registry cache, trying to recreate it

Caused by:
    Database already open. Cannot acquire lock.
error: failed to lookup for `snforge_std ^0.30.0` in registry: registry+https://scarbs.xyz/

Caused by:
    0: failed to remove local registry cache
    1: failed to remove file `/home/runner/.cache/scarb/registry/cache/scarbs.xyz-9djtpev4jug5q.v1.redb`
    2: No such file or directory (os error 2)
Error: [ERROR] error: could not gather project metadata from Scarb due to previous error: `scarb metadata` exited with error

```

Do you have idea how to fix it?

**Comment by THenry14** [comment]
> Seems that when we use forge from registry sometimes CI is failing:
> 
> ```
> warn: failed to open local registry cache, trying to recreate it
> 
> Caused by:
>     Database already open. Cannot acquire lock.
> error: failed to lookup for `snforge_std ^0.30.0` in registry: registry+https://scarbs.xyz/
> 
> Caused by:
>     0: failed to remove local registry cache
>     1: failed to remove file `/home/runner/.cache/scarb/registry/cache/scarbs.xyz-9djtpev4jug5q.v1.redb`
>     2: No such file or directory (os error 2)
> Error: [ERROR] error: could not gather project metadata from Scarb due to previous error: `scarb metadata` exited with error
> ```
> 
> Do you have idea how to fix it (as you worked previously on registry)?

Hmm, this looks like a scarb issue potentially, maybe a race condition. My wild guess is locks are at fault, so you could try [to set a different cache per problematic test(s) ](https://docs.swmansion.com/scarb/docs/reference/global-directories.html#cache-directory) but this is suboptimal for large number of tests probably. 

I know this is a known issue in scarb, but not sure if any progress to fix this properly is being made tbh 🤔

**Comment by ksew1** [comment]
> > Seems that when we use forge from registry sometimes CI is failing:
> > ```
> > warn: failed to open local registry cache, trying to recreate it
> > 
> > Caused by:
> >     Database already open. Cannot acquire lock.
> > error: failed to lookup for `snforge_std ^0.30.0` in registry: registry+https://scarbs.xyz/
> > 
> > Caused by:
> >     0: failed to remove local registry cache
> >     1: failed to remove file `/home/runner/.cache/scarb/registry/cache/scarbs.xyz-9djtpev4jug5q.v1.redb`
> >     2: No such file or directory (os error 2)
> > Error: [ERROR] error: could not gather project metadata from Scarb due to previous error: `scarb metadata` exited with error
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Do you have idea how to fix it (as you worked previously on registry)?
> 
> Hmm, this looks like a scarb issue potentially, maybe a race condition. My wild guess is locks are at fault, so you could try [to set a different cache per problematic test(s) ](https://docs.swmansion.com/scarb/docs/reference/global-directories.html#cache-directory) but this is suboptimal for large number of tests probably.
> 
> I know this is a known issue in scarb, but not sure if any progress to fix this properly is being made tbh 🤔
> 
> **Edit: The fix is already on master, and will be released in scarb 2.9.1** :D [software-mansion/scarb#1712](https://github.com/software-mansion/scarb/pull/1712)



It seems that tests are failing with the new Scarb 2.9.1. Until I investigate this, let's use Git instead of the registry. I have created a PR that fixes the issue here: https://github.com/software-mansion/cairo-coverage/pull/112

**Pull request merged**